src/Pages/Dashboard/ShowHistoryGraph.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {PureComponent} from "react";
+import {Paper, Typography, FormControl, InputLabel, Select, MenuItem, Grid, withStyles} from "@material-ui/core"
+import {historyStyle} from "./dashboardStyle";
+
+class ShowHistoryGraph extends PureComponent {
+    constructor(props) {
```

Comment:
If you're not setting any state or class-specific fields, you won't need this constructor since it'll call it implicitly.